playground/src/Designer.tsx

```diff
@@ -36,7 +36,14 @@ function App() {
         designer.current = new Designer({
           domContainer: designerRef.current,
           template,
-          options: { font },
+          options: {
+            font,
+            theme: {
```

Comment:
We can use like this.

Customize Design Token: https://ant.design/docs/react/customize-theme#customize-design-token